packages/components/src/components/multi-select/multi-select/multi-select-utils.ts

```diff
@@ -0,0 +1,166 @@
+import { FormState } from '../../../utils/form/form-state';
+import {
+  determineDropdownDirection,
+  SelectDropdownDirection,
+  SelectDropdownDirectionInternal,
+} from '../../../utils/select/select-dropdown';
+import { Theme } from '../../../utils';
+import { forceUpdate } from '@stencil/core';
+import { MultiSelectOptionInternalHTMLProps } from '../multi-select-option/multi-select-option-utils';
+
+export type MultiSelectState = FormState;
+export type MultiSelectDropdownDirection = SelectDropdownDirection;
+
+export const syncMultiSelectOptionProps = (options: HTMLPMultiSelectOptionElement[], theme: Theme): void => {
+  options.forEach((item: HTMLElement & MultiSelectOptionInternalHTMLProps) => {
+    item.theme = theme;
+    forceUpdate(item);
+  });
+};
+
+export const syncNativeSelect = (
```

Comment:
Perhaps this should be more like `initNativeSelect` which also takes care of creating it and returning the element instance which then can be saved in the component instance and also used for later syncing. Then we wouldn't need the `!host.querySelector('SELECT')` within this function, too.